N/NCDatasets/Versions.toml

```diff
@@ -135,9 +135,11 @@ git-tree-sha1 = "afd015e81e60cfbdba04ef59bcdc80e18bd613cd"
 
 ["0.12.15"]
 git-tree-sha1 = "6450a225f79309a21397e83563287978d5f875a1"
+yanked = true
 
 ["0.12.16"]
 git-tree-sha1 = "e201ed836f4486d0a5f593e68b7621b2e24237c5"
 
 ["0.12.17"]
 git-tree-sha1 = "4263c4220f22e20729329838bf7e94a49d1ac32f"
+
```

Comment:
Now there's an _extra_ new line at the end of the file, before there was none.  There should be _exactly_ one, no more, no less.
```suggestion
```